lib/components/victory-brush-container.d.ts

```diff
@@ -0,0 +1,5 @@
+import { VictoryBrushContainer } from "victory-brush-container";
+
+export const brushContainerMixin: (base: any) => VictoryBrushContainer;
+
+export default VictoryZoomContainer;
```

Comment:
I think this should be `export default VictoryBrushContainer`

Reply:
Ha, yes it should be!  I'll fix right away

Reply:
Fixed